### ISSUE TITLE
`virtual` keyword outdated for static interface methods

### ISSUE THREAD

**Issue by itsBuggingMe** [dotnet/docs#42852]
### Type of issue

Outdated article

### Description

The docs state that:
> It is an error to use the `virtual` modifier on a static property.
> You cannot use the `virtual` modifier with the `static`, `abstract`, `private`, or `override` modifiers.

This is no longer true due to virtual static interface methods. For example, this compiles on language versions 11.0 and above:

```csharp
public interface IInterface
{
    virtual static void StaticMethod() { }
    virtual static int StaticProperty { get; set; }
}
```

### Page URL

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/virtual

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/keywords/virtual.md

### Document Version Independent Id

6a4f3ec1-e2eb-e046-d840-1c442276b3fb

### Article author

@BillWagner

### Metadata

* ID: 89022fe7-b2e3-5fb5-4930-d6bc5284081a 
* Service: **dotnet-csharp**
* Sub-service: **lang-reference**

**Comment by BillWagner** [comment]
Thanks for pointing this out @itsBuggingMe 

I've added it to our backlog to fix when we next update this article.

**Comment by bigboybamo** [comment]
Hi @BillWagner I opened a PR I think might fix this #42866
